examples/lsm6ds_full_test.py

```diff
@@ -0,0 +1,29 @@
+import time
+import board
+import busio
+#pylint:disable=no-member,unused-import
+from adafruit_lsm6ds import LSM6DS33, LSM6DSOX, ISM330DHCT, Rate, AccelRange, GyroRange
+
+i2c = busio.I2C(board.SCL, board.SDA)
+
+sensor = LSM6DS33(i2c)
+#sensor = LSM6DSOX(i2c)
+#sensor = ISM330DHCT(i2c)
+
+sensor.accelerometer_range = AccelRange.RANGE_8G
+print("Accelerometer range set to: %d G"%AccelRange.string[sensor.accelerometer_range])
+
+sensor.gyro_range = GyroRange.RANGE_2000_DPS
+print("Gyro range set to: %d DPS"%GyroRange.string[sensor.gyro_range])
```

Comment:
ditto for all % strings, add a space between % and left/right halves for readability

Reply:
done